src/components/Home-User-Meta.js

```diff
@@ -121,7 +127,7 @@ class HomeUserMeta extends Component {
           { this.userIsHirable() }
         </div>
         <div className="User-Meta-profile">
-          <img className="User-Meta-profile-picture" src={ this.userImg() } alt="user" />
+          <a href={ this.userImgUrl() } target="_blank"><img className="User-Meta-profile-picture" src={ this.userImg() } alt="user" /></a>
```

Comment:
Hi @dansegliode is the `className` being used? If not please remove

Reply:
yes, it was there before

Reply:
Sorry I did not see this :)

Reply:
Read comment below.